server/game/effects.js

```diff
@@ -59,12 +59,14 @@ const Effects = {
     alert: () => EffectBuilder.card.static('alert'),
     bound: () => EffectBuilder.card.static('bound'),
     canGuard: () => EffectBuilder.card.static('canGuard'),
-    concealed: () => EffectBuilder.card.static('concealed'),
+    cannotBeSpellTarget: () => EffectBuilder.card.static('cannotBeSpellTarget'),
+    cannotBeAbilityTarget: () => EffectBuilder.card.static('cannotBeAbilityTarget'),
+    cannotBeDicePowerTarget: () => EffectBuilder.card.static('cannotBeDicePowerTarget'),
     cannotBeAttackTarget: () => EffectBuilder.card.static('cannotBeAttackTarget'),
+    cannotBeReactionTarget: () => EffectBuilder.card.static('cannotBeReactionTarget'),
+    cannotBeAffectedBySpells: () => EffectBuilder.card.static('cannotBeAffectedBySpells'), //implement this to fix issue #613
```

Comment:
note this, which is not yet checked for anywhere (but is given to all spellguard units). That leads to the bugs in #613